sdk/forms/test/forms.spec.ts

```diff
@@ -0,0 +1,82 @@
+
+import { test, expect } from '@playwright/test';
+import { resolve, dirname } from 'path'
+import { fileURLToPath } from 'url'
+import {createServer, ViteDevServer} from 'vite'
+import {AddressInfo} from "node:net";
+
+// @ts-ignore
+const rootDir = resolve(dirname(fileURLToPath(import.meta.url)), '..')
+
+let url: string;
+let server: ViteDevServer;
+
+test.beforeAll(async () => {
+  server = await createServer({
+    // any valid user config options, plus `mode` and `configFile`
+    configFile: resolve(rootDir, 'vite.config.ts'),
+    root: rootDir,
+  })
+  await server.listen()
+  const {port} = (server?.httpServer?.address() as AddressInfo)
+  url = `http://localhost:${port}/`
+})
+
+test.afterAll(async () => {
+  await server.close()
+})
+
+test('submitting the forms tokenize the data', async ({ page }) => {
+  await page.goto(url);
+
+  const cardNumberInput =page.getByLabel('Card Number')
```

Comment:
Formatter?

Reply:
None of the projects in this repo has prettier or even eslint at the moment.
I'll manually format it and we should add it later for all projects.